### PR TITLE
Prepare metadata.txt for release

### PR DESCRIPTION
**Pull request by ghtmtt**

### CODE REVIEW

File: metadata.txt [hunk 1]
```diff
@@ -12,8 +12,8 @@ email=matteo.ghetta@faunalia.eu
```
Comment (MattF-NSIDC) [inline metadata.txt:12]: Hey @ghtmtt can you change the `name` field on line `6` to `QGreenland Custom`? We decided that we are going to brand the plugin as `QGreenland Custom` and the zip package as `QGreenland Core`.

**Review by trey-stafford — CHANGES_REQUESTED** [review]
Actually, before we merge this and do a release, let's change the plugin name to `QGreenland Custom`.

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]

**Comment by ghtmtt** [comment]
> We decided that we are going to brand the plugin as QGreenland Custom and the zip package as QGreenland Core

what do you exactly mean by naming the zip package? We only need to create the zip package to upload the plugin to the QGIS server, the user won't ever see the zip package...

**Comment by trey-stafford** [comment]
> > We decided that we are going to brand the plugin as QGreenland Custom and the zip package as QGreenland Core
> 
> what do you exactly mean by naming the zip package? We only need to create the zip package to upload the plugin to the QGIS server, the user won't ever see the zip package...

Hey @ghtmtt sorry we were not clear on this. The zip package we are referring to is the QGreenland zip package that comes pre-configured with a project file and data, which is already available on our website, https://qgreenland.org/download.

Nothing in this repo needs to be branded as 'QGreenland Core'. We just want the name of the plugin to be renamed to 'QGreenland Custom'.

**Review by MattF-NSIDC — APPROVED** [review]

**Comment by ghtmtt** [comment]
@trey-stafford got it. So the changes made in the current `metadata.txt` file are enough for this release right?

**Review by trey-stafford — APPROVED** [review]

**Comment by trey-stafford** [comment]
> @trey-stafford got it. So the changes made in the current `metadata.txt` file are enough for this release right?

Yes! Thanks, @ghtmtt!

**Pull request merged**